CHANGES.rst

```diff
@@ -2,6 +2,16 @@ Margarita
 
 Changes - always add to the top.
 
+v 1.7.0 (unreleased)
+--------------------
+
+* Allow specifying the branch to deploy in the 'branch' pillar variable.
+    - Most projects should now be able to specify repo + url just once,
+      in `project.sls`.
+    - With this structure, the branch to deploy can now easily be overridden
+      from the command line. See Salt docs:
+      https://docs.saltstack.com/en/latest/topics/pillar/#set-pillar-data-at-the-command-line
```

Comment:
Maybe add an example command line here?  (Or if we have an example in the developer docs, link there?)